ext/native-decls/SetNuiZindex.md

```diff
@@ -0,0 +1,13 @@
+---
+ns: CFX
+apiset: client
+---
+## SET_NUI_ZINDEX
```

Comment:
Nit: It's good practice to have a description for the native, even if its name makes its purpose obvious.

Reply:
I changed it